website/docs/SeleniumGrid.md

```diff
@@ -0,0 +1,109 @@
+---
+id: seleniumgrid
+title: Selenium Grid
+---
+
+You can use WebdriverIO with your existing Selenium Grid instance. To connect your tests to Selenium Grid, you just need to update the options in your test runner configurations.
+
+Here is a code snippet from sample wdio.conf.ts.
+
+```ts
+// wdio.conf.ts
+
+export const config: Options.Testrunner = {
+    runner: 'local',
+// ...
+    protocol: 'https',
+    hostname: 'yourseleniumgridhost.yourdomain.com',
+    port: 443,
+    path: '/wd/hub',
+// ...
```

Comment:
The `runner` option is not relevant for this example, let's keep the example simple
```suggestion
    // ...
    protocol: 'https',
    hostname: 'yourseleniumgridhost.yourdomain.com',
    port: 443,
    path: '/wd/hub',
    // ...
```